pkg/minishift/registration/utils.go

```diff
@@ -30,7 +31,7 @@ type RegistrationParameters struct {
 }
 
 // Register host VM
-func RegisterHostVM(host *host.Host, param *RegistrationParameters) error {
+func RegisterHostVM(host *host.Host, param *RegistrationParameters, proxyInfo string) error {
```

Comment:
I definitely thought about this option. But decided against it because proxy is just not required by registration but also for docker and OpenShift. So if an information is shared between Red Hat subscription manager , docker and OpenShift it is not logical to put it RegistrationParameters structure.